src/document_to_podcast/cli.py

```diff
@@ -27,9 +26,9 @@
 def document_to_podcast(
     input_file: str | None = None,
     output_folder: str | None = None,
-    text_to_text_model: str = "allenai/OLMoE-1B-7B-0924-Instruct-GGUF/olmoe-1b-7b-0924-instruct-q8_0.gguf",
+    text_to_text_model: str = "bartowski/Qwen2.5-3B-Instruct-GGUF/Qwen2.5-3B-Instruct-f16.gguf",
     text_to_text_prompt: str = DEFAULT_PROMPT,
-    text_to_speech_model: TTS_LOADERS = "OuteAI/OuteTTS-0.2-500M-GGUF/OuteTTS-0.2-500M-FP16.gguf",
+    text_to_speech_model: str = "OuteAI/OuteTTS-0.2-500M-GGUF/OuteTTS-0.2-500M-FP16.gguf",
```

Comment:
Is there a reason you removed the `TTS_LOADERS`?

Reply:
It is an invalid (and wrong) type hint.
You can't type with a variable and the expected type is a string, not a dict. To properly type this we would need to create a Literal from the TTS_LOADERS keys

Reply:
I see, makes sense.